deploy/index-image/community-kubevirt-hyperconverged/1.4.0/hco00.crd.yaml

```diff
@@ -75,6 +101,25 @@ spec:
                     description: Enable HyperV strict host checking for HyperV enlightenments
                       Defaults to true, even when HyperConvergedFeatureGates is empty
                     type: boolean
+                  liveMigration:
+                    default: true
+                    description: Enables VMIs to be live migrated. Without this, migrations
+                      are not possible and will be blocked
```

Comment:
Why would I, as sysadmin, like to disable `liveMigration`? I believe that this should be hard-coded to `true`.